addons/html_editor/static/src/utils/selection.js

```diff
@@ -369,25 +369,23 @@ export function getDeepRange(editable, { range, sel, splitText, select, correctT
  * selection.
  *
  * @param {Node} editable
+ * @param {Selection} selection
  * @returns {Node[]}
  */
-export function getTraversedNodes(editable, range = getDeepRange(editable)) {
+export function getTraversedNodes(editable, selection) {
```

Comment:
I should consider moving this function (and `getTraversedBlocks`) to the selection plugin...